webui/react/src/pages/TrialDetails/Profiles/Charts/SystemMetricChart.tsx

```diff
@@ -102,7 +103,11 @@ const SystemMetricChart: React.FC<ChartProps> = ({ trial }) => {
         xAxis={XAxisDomain.Time}
         xLabel="Time"
         yLabel={yLabel}
-        yTickValues={getScientificNotationTickValues}
+        yTickValues={
+          yLabel.substring(0, 5).toLowerCase() === 'bytes'
```

Comment:
I prefer using regex for situations like this, which would look something like `/^bytes/i.test(yLabel)`, but it's not a big deal